### PR TITLE
(wip) New setup section

### PR DESCRIPTION
**Pull request by Sims24**
## Description

New setup section to manage all supported package manager.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [x] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 1]
```diff
@@ -53,33 +53,73 @@ The centreon-engine user performs a SSH connection to a remote system user. This
 
 ## Setup
 
+### Monitoring Pack
+
+If the platform uses an *online* license, you can skip the package installation
+instruction below as it is not required to have the Pack displayed within the
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md:59]:
```suggestion
instruction below as it is not required to have the pack displayed within the 
```

File: pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 2]
```diff
@@ -53,33 +53,73 @@ The centreon-engine user performs a SSH connection to a remote system user. This
...
+### Plugin
+
+Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature. 
+When this features is enabled, you can skip the installation part below. 
+
+You still have to manually install the Plugin on the Poller(s) when: 
+- Automatic plugin installation is turned off
+- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet
+
+Use amongst commands below according to your operating system's package manager:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md:101]:
```suggestion
Use the commands below according to your operating system's package manager: 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 3]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*. L'installation d'un paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:58]:
```suggestion
Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 4]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*. L'installation d'un paquet 
+n'est pas requise pour voir apparaître le Pack dans le menu **Configuration > Plugin Packs > Gestionnaire**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:59]:
```suggestion
n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 5]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*. L'installation d'un paquet 
+n'est pas requise pour voir apparaître le Pack dans le menu **Configuration > Plugin Packs > Gestionnaire** 
+
+Au contraire, si la plateforme utilise une licence *online*, installer le paquet 
+sur le **serveur central** via la commande correspondante au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:62]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 6]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
+```bash title='RHEL, Oracle Linux, Alma Linux version <= 8'
+yum install centreon-pack-applications-drbd-ssh
+```
 
 </TabItem>
-<TabItem value="Offline License" label="Offline License">
+</Tabs>
 
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRBD :
+Quelque soit le type de la licence (*online* ou *offline*), installer le Pack *DRBD SSH*
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:89]:
```suggestion
Quel que soit le type de la licence (*online* ou *offline*), installez le Pack **DRBD SSH** 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 7]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
 
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRBD :
+Quelque soit le type de la licence (*online* ou *offline*), installer le Pack *DRBD SSH* 
+depuis l'interface web et le menu **Configuration > Plugin Packs > Gestionnaire**. 
 
-```bash
-yum install centreon-plugin-Applications-Drbd-Ssh.noarch
+### Plugin
+
+A partir de Centreon 22.04, il est possible de demander le déploiement automatique
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:94]:
```suggestion
À partir de Centreon 22.04, il est possible de demander le déploiement automatique 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 8]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRBD :
+Quelque soit le type de la licence (*online* ou *offline*), installer le Pack *DRBD SSH* 
+depuis l'interface web et le menu **Configuration > Plugin Packs > Gestionnaire**. 
 
-```bash
-yum install centreon-plugin-Applications-Drbd-Ssh.noarch
+### Plugin
+
+A partir de Centreon 22.04, il est possible de demander le déploiement automatique 
+du plugin lors de l'utilisation d'un Pack. Si cette fonctionnalité est activée, et
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:95]:
```suggestion
du plugin lors de l'utilisation d'un pack. Si cette fonctionnalité est activée, et 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 9]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*. L'installation d'un paquet 
+n'est pas requise pour voir apparaître le Pack dans le menu **Configuration > Plugin Packs > Gestionnaire** 
+
+Au contraire, si la plateforme utilise une licence *online*, installer le paquet 
+sur le **serveur central** via la commande correspondante au gestionnaire de paquet 
+associé à sa distribution:
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:63]: This new presentation is not user-friendly

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 10]
```diff
@@ -53,33 +53,73 @@ The centreon-engine user performs a SSH connection to a remote system user. This
...
+If the platform uses an *online* license, you can skip the package installation
+instruction below as it is not required to have the pack displayed within the 
+**Configuration > Plugin Packs > Manager** menu. 
+
+If the platform uses an *offline* license, install the package on the **central server**
+with the command corresponding to the operating system's package manager: 
 
-1. Install the Centreon Plugin on every Poller monitoring DRBD resources:
+<Tabs groupId="sync">
+<TabItem value="RHEL, Oracle Linux, Alma Linux version >= 8" label="RHEL, Oracle Linux, Alma Linux version >= 8">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-drbd-ssh.md:66]: Non mais en fait faut mettre explicitement les numéros des versions supportées, sinon ça sous-entend qu'on supporte tout au-dessus ou au-dessous de la version 8, donc c'est pas bon

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 11]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
+Au contraire, si la plateforme utilise une licence *offline*, installer le paquet 
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet 
+associé à sa distribution: 
+
 <Tabs groupId="sync">
-<TabItem value="Online License" label="Online License">
+<TabItem value="RHEL, Oracle Linux, Alma Linux 8" label="RHEL, Oracle Linux, Alma Linux 8">
 
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRDB :
+```bash title='RHEL, Oracle Linux, Alma Linux version >= 8'
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:68]: Same comment as the previous one... And the tabs are enough, you don't need a title there.

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 12]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet 
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installer le paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:61]:
```suggestion
Au contraire, si la plateforme utilise une licence *offline*, installez le paquet 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 13]
```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
...
-<TabItem value="Offline License" label="Offline License">
+</Tabs>
+
+Quel que soit le type de la licence (*online* ou *offline*), installez le Pack **DRBD SSH** 
+depuis l'interface web et le menu **Configuration > Plugin Packs > Gestionnaire**. 
+
+### Plugin
 
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRBD :
+À partir de Centreon 22.04, il est possible de demander le déploiement automatique
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md:94]: link?

**Comment by Sims24** [comment]
Will be outdated.

**Pull request closed without merging**